kover.gradle

```diff
@@ -0,0 +1,24 @@
+apply from: '../jacoco-config.gradle'
```

Comment:
do we also need to delete jacoco-config.gradle?

Reply:
Yeah I would just pull the excludes list into this file and delete that

Reply:
Also delete jacoco.gradle